editor/editor_build_profile.cpp

```diff
@@ -405,7 +737,11 @@ void EditorBuildProfileManager::_profile_action(int p_action) {
 			confirm_dialog->popup_centered();
 		} break;
 		case ACTION_DETECT: {
-			confirm_dialog->set_text(TTR("This will scan all files in the current project to detect used classes."));
+			String text = TTR("This will scan all files in the current project to detect used classes.\nNote that the first scan may take a while, specially in larger projects.");
+#ifdef MODULE_MONO_ENABLED
+			text += "\n\n" + TTR("Warning: Class detection for C# scripts is not currently available, and such files will be ignored.");
+#endif // MODULE_MONO_ENABLED
```

Comment:
FYI @godotengine/dotnet - would be good to implement as a follow-up.

Reply:
Oh believe me, I tried, and it's currently not possible in a way that makes it worth it. You can ask @raulsntos for more details.